src/ui/actions/comments.ts

```diff
@@ -228,9 +234,14 @@ export function seekToComment(item: Comment | Reply): UIThunkAction {
 
 async function getCurrentPauseSourceLocation(
   ThreadFront: typeof ThreadFrontType,
+  replayClient: ReplayClientInterface,
   getState: () => UIState
 ) {
-  const frame = (await ThreadFront.currentPause?.getFrames())?.[0];
+  const pauseId = getPauseId(getState());
+  if (!pauseId) {
+    return;
+  }
+  const frame = (await getFramesAsync(replayClient, pauseId))?.[0];
```

Comment:
Is this (assuming the first frame) safe/correct?

Reply:
I think so. This is used when adding comments to the screenshot or in the logpoint panel - in both cases I think the comment should be associated with the top frame (and if there is no top frame we bail out).